6-Typescript/server/models/TodoDAODataAPI.ts

```diff
@@ -1,49 +1,65 @@
-import mongoose from 'mongoose';
-import { Schema, Model, model } from 'mongoose';
-import { ITodo } from '../interfaces/ITodo';
-import { ITodoService } from '../interfaces/ITodoService';
+import { Schema, Model, model } from "mongoose";
+import { ITodo } from "../interfaces/interfaces";
+import { ITodoService } from "../interfaces/interfaces";
 const todoScheme: Schema = new Schema<ITodo>({
     todoText: {
         type: String,
-        required: true
+        required: true,
     },
     todoId: {
         type: String,
-        required: true
+        required: true,
     },
     checked: {
         type: Boolean,
-        required: true
+        required: true,
     },
     userId: {
         type: String,
-        required: true
-    }
-})
+        required: true,
+    },
+});
 export class TodoDAODataAPI implements ITodoService {
-    Todo: Model<ITodo>
+    Todo: Model<ITodo>;
     constructor() {
-        this.Todo = model('Todo', todoScheme)
-    }
-    modelGetTodoById = async (userId: string, todoId: string): Promise<ITodo | null> => {
-        return await this.Todo.where({'userId': userId}).findOne({'todoId': todoId}, {"todoText":1,"todoId":1, "checked": 1, "userId": 1, "_id":0}).lean(true);            
-    }
-    
-    modelGetAllTodos = async (userId: string): Promise<ITodo[] | null>  => {
-        return await this.Todo.find({userId: userId}, {"todoText":1,"todoId":1, "checked": 1, "userId": 1, "_id":0})     
+        this.Todo = model("Todo", todoScheme);
     }
-    
+    modelGetTodoById = async (
+        userId: string,
+        todoId: string
+    ): Promise<ITodo | null> => {
+        return await this.Todo.where({ userId: userId })
+            .findOne(
+                { todoId: todoId },
+                { todoText: 1, todoId: 1, checked: 1, userId: 1, _id: 0 }
+            )
+            .lean(true);
+    };
+
+    modelGetAllTodos = async (userId: string): Promise<ITodo[] | null> => {
+        return await this.Todo.find(
+            { userId: userId },
+            { todoText: 1, todoId: 1, checked: 1, userId: 1, _id: 0 }
```

Comment:
what? what are those values? (`1, 1, 1, 1, 0`) ?
they mean nothing to me as a reviewer (and in the future - for your teammates that will need to understand and collaborate to your code )

Reply:
this is how you write in mongo what values in the json i want and dont. 1 is yes and 0 is no. 
I can create an enum for that as well.

Reply:
enum sounds better :)